### PR TITLE
Update tags

### PR DESCRIPTION
**Pull request by QuanWanxx**
Update Azure Tags from ```{"SynapseLinkSolution": "fhir"}``` to ```{"FhirAnalyticsPipeline": "FhirToDatalake", "FhirSchemaVersion": "v0.3.0"}```

```FhirToDatalake``` indicates our solution type of pipeline.
```v0.3.0``` is our planning release tag of FhirAnalyticsPipeline schemas.

### CODE REVIEW

File: FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
         "functionAppName": "[variables('appName')]",
         "hostingPlanName": "[variables('appName')]",
         "applicationInsightsName": "[concat('AppInsights-', variables('appName'))]",
-        "synapseLinkTag": "fhir",
+        "resourceTags": {"FhirToSynapsePipeline": "FunctionV0.3.0", "FhirSchema": "R4_V1"},
```
Comment (BoyaWu10) [inline FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json:95]: Nit: Maybe we can remove the `Function` in the first tag?

Comment (QuanWanxx) [reply FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json:95]: FhirToCDM uses Azure DataFactory as the agent to fetch data from fhir server, just want to distinguish them in cases we may improve that.

Comment (moria97) [reply FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json:95]: What about add a `_` between function and v0.3.0? like `Function_v0.3.0`

Comment (QuanWanxx) [reply FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json]: Sure, updated

**Review by qiwjin — APPROVED** [review]

**Pull request merged**